From 7c64ea48329b9a8fa4b1100c10f7413da800a83d Mon Sep 17 00:00:00 2001
From: "Reynaldo H. Verdejo Pinochet" <reynaldo@osg.samsung.com>
Date: Thu, 17 Nov 2016 11:19:54 -0800
Subject: [PATCH 08/15] wayland: avoid prefix clash in combined pkg-config
 check

Adittionally, remove leftover WAYLAND_DRM_ include flags var

Fixes building of ext/wayland/* broken by dfd86de67a8c04f6d1

Related to:

https://bugzilla.gnome.org/show_bug.cgi?id=773927
---
 ext/wayland/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ext/wayland/Makefile.am b/ext/wayland/Makefile.am
index c156fcc..8442dbd 100644
--- a/ext/wayland/Makefile.am
+++ b/ext/wayland/Makefile.am
@@ -23,8 +23,7 @@ libgstwaylandsink_la_CFLAGS = \
 	$(GST_PLUGINS_BAD_CFLAGS) \
 	$(GST_PLUGINS_BASE_CFLAGS) \
 	$(GST_CFLAGS) \
-	$(WAYLAND_CFLAGS) \
-	$(WAYLAND_DRM_CFLAGS)
+	$(WAYLAND_CFLAGS)
 libgstwaylandsink_la_LIBADD = \
 	$(top_builddir)/gst-libs/gst/wayland/libgstwayland-$(GST_API_VERSION).la \
 	$(GST_PLUGINS_BASE_LIBS) \
-- 
2.7.4

